ivy/functional/backends/paddle/experimental/layers.py

```diff
@@ -257,7 +257,17 @@ def dropout1d(
     data_format: str = "NWC",
     out: Optional[paddle.Tensor] = None,
 ) -> paddle.Tensor:
-    raise IvyNotImplementedException()
+    if training:
+        is_batched = len(x.shape) == 3
+        if data_format == "NCW":
+            perm = (0, 2, 1) if is_batched else (1, 0)
+            x = paddle.transpose(x, perm)
+        res = paddle.nn.functional.dropout(x, prob)
+        if data_format == "NCW":
+            res = paddle.transpose(res, perm)
+    else:
+        res = x
+    return res
```

Comment:
these are suggested changes, no need to check for batched input and creating a permutation since we are swapping the last two axes which can be done using `swapaxes` or `moveaxis` in our case
also paddle dropout has a `training` argument which removes the need for the if condition

```suggestion
    if data_format == "NCW":
        x = paddle.moveaxis(x, -2,-1)
    res = paddle.nn.functional.dropout(x, p=prob,training=training)
    if data_format == "NCW":
        res = paddle.moveaxis(res, -2,-1)
    return res
```